README.md

```diff
@@ -144,7 +137,7 @@ $ oc create namespace fake-cluster-namespace
 $ oc apply -f hack/clusterdeployment/fake-clusterdeployment.yml
 ```
 
-If present, set `status.installed` to true.
+If present, set `spec.installed` to true.
```

Comment:
why was this moved?

Reply:
Good question! The change here in the README that I've made is just to reflect reality. It looks like it was changed deliberately in the code, but the information here in the README was forgotten.

As to why it was moved in the code, I'm not sure. It looks like it was done in #74 for some reason. The title of that pull request is "Status.Installed is deprecated, we should use Spec.Installed", maybe that means something to somebody?

Actually, given that this operator doesn't own the ClusterDeployment CRD, looking in the hive project is where we should be looking. This change seems to have all the details: https://github.com/openshift/hive/pull/526.